README.md

```diff
@@ -36,581 +34,25 @@ The following set of features makes Artipie unique among all others:
     [HuaweiCloud OBS](https://www.huaweicloud.com/en-us/product/obs.html) etc.
   * Its quality of Java code is extraordinary high :)
 
-The fastest way to start using Artipie is via
```

Comment:
@Sammers21 hm, are we going to leave some short instruction how to start artipie in the main readme?

Reply:
@olenagerasimova, no, the user will go to the examples section for a demo.

Reply:
@Sammers21 I'm not sure that a good idea as 
a) my reaction to such readmes usually is "ok, you think you are awesome, but how the hell do I use it"?
b) it requires a lot of actions: open another readme, process table of content, etc
c) we are adding auto-generation of the simple config files => instruction to start artipie with trivial configuration should be very short and perfectly feet into this readme

Reply:
> @Sammers21 I'm not sure that a good idea as
> a) my reaction to such readmes usually is "ok, you think you are awesome, but how the hell do I use it"?

And then you realize that there is an Example section, you go to the section and enjoy.

Reply:
@olenagerasimova, 

> b) it requires a lot of actions: open another readme, process table of content, etc

Since the navigation is extremely simple and clear it would not take much efforts

> c) we are adding auto-generation of the simple config files => instruction to start artipie with trivial configuration should be very short and perfectly feet into this readme

I think that a README file should contain only the most important information, if user will need an example, he will find it really quickly.